use-cases/ai-agent.ipynb

```diff
@@ -6,16 +6,16 @@
     "id": "3kMALXaMv-MS"
    },
    "source": [
-    "# Build an AI Agent with MongoDB Atlas"
+    "# Build an AI Agent with MongoDB"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook is a companion to the [Build AI Agents with MongoDB Atlas](https://www.mongodb.com/docs/atlas/atlas-vector-search/build-agents) page. For a more traditional Python development example and detailed explanations of the code, refer to the tutorial on the page.\n",
+    "This notebook is a companion to the [Build AI Agents with MongoDB](https://www.mongodb.com/docs/atlas/atlas-vector-search/build-agents) page. For a more traditional Python development example and detailed explanations of the code, refer to the tutorial on the page.\n",
```

Comment:
https://www.mongodb.com/docs/atlas/atlas-vector-search/build-agents this link 404s for me